poc/vdaf_mastic.py

```diff
@@ -188,7 +195,8 @@ def prep_shares_to_prep(cls, agg_param, prep_shares):
         (level_proof_0, verifier_share_0) = prep_shares[0]
         (level_proof_1, verifier_share_1) = prep_shares[1]
         if level_proof_0 != level_proof_1:
-            raise Exception('output vector is not one-hot')
+            raise Exception('output vector is not one-hot or VIDPF output is \
+                            inconsistent between levels')
```

Comment:
Otherwise this string contains a bunch of empty space
```suggestion
            raise Exception('output vector is not one-hot or VIDPF output is ' \
                            'inconsistent between levels')
```